tool/python/singa/utils/utility.py

```diff
@@ -82,3 +83,9 @@ def setval(proto, **kwargs):
                     fattr.MergeFrom(val)
                 else:
                     setattr(proto, key, val)
+
+def swap32(x):
```

Comment:
If this function is only used in parsing MNIST dataset, then we can move it into that file.